devlab/tests/test_verify_dst_del_tenant_resources.py

```diff
@@ -0,0 +1,190 @@
+import tests.config as config
+import tests.functional_test as functional_test
+
+TEST_TENANT_NAME = 'tenant3'
```

Comment:
I think you can remove this constant and use list of deleted tenants from config.py